src/wp-admin/includes/template.php

```diff
@@ -2270,17 +2270,19 @@ function _post_states( $post, $display = true ) {
 	 *
 	 * @since 6.9.0
 	 *
-	 * @param string   $post_states_string The post states HTML string.
-	 * @param string[] $post_states        The post states.
-	 * @param WP_Post  $post               The current post object.
+	 * @param string                 $post_states_html All relevant post sta60986tes combined into an HTML string for display.
+	 *                                                 E.g. `&mdash; <span class='post-state'>Draft, </span><span class='post-state'>Sticky</span>`.
+	 * @param string<string, string> $post_states      A mapping of post state slugs to translated post state labels.
+	 *                                                 E.g. `array( 'draft' => 'Draft', 'sticky' => 'Sticky' )`.
```

Comment:
while this is great already, do you think it could help or possibly add confusion if we showed the translation function?

```php
 * E.g. `array( 'draft' => __( 'Draft' ), … )`
```